var/spack/repos/builtin/packages/hdf5-vol-async/package.py

```diff
@@ -27,10 +27,7 @@ class Hdf5VolAsync(CMakePackage):
     depends_on("hdf5@1.13: +mpi +threadsafe")
 
     def setup_run_environment(self, env):
-        env.set("HDF5_PLUGIN_PATH", self.spec.prefix.lib)
-        vol_connector = "async under_vol=0;under_info=[]"
```

Comment:
Why are we removing this? When applications load the ASYNC-VOL (spack load hdf5-vol-async), they want to use the connector. So shouldn't spack handle that setting instead of having users set it?

Reply:
(marking resolved to keep discussion on the same thread above)